### PR TITLE
Make sure the ports are unique when running the multinode test

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

File: Framework/script/o2-qc-multinode-test.sh [hunk 1]
```diff
@@ -42,29 +49,20 @@ function delete_data() {
...
-if [ -z "$UNIQUE_PORT_1" ]
-then
-  echo "UNIQUE_PORT_1 must be set when calling o2-qc-multinode-test.sh"
-  exit 1
-fi
+# mkdir in /tmp is guaranteed to be atomic
+mkdir /tmp/${UNIQUE_TEST_NAME} || { echo "Concurrent usage of the same port ${UNIQUE_PORT_1} detected, exiting"; exit 1; }
+pushd /tmp/${UNIQUE_TEST_NAME}
+
+UNIQUE_PORT_2=$((UNIQUE_PORT_1+1))
```
Comment (ktf) [inline Framework/script/o2-qc-multinode-test.sh:62]: To avoid conflicts, the cmake code should be changed to have only odd / even ports.

**Comment by ktf** [comment]
Well, at least we can reproduce the issue.

**Comment by ktf** [comment]
For the record, one thing I noticed on my mac is that the port is always the same...

**Comment by knopers8** [comment]
It is generated at cmake time, not execution time. Do you mean it is the same when running cmake over and over?

**Comment by ktf** [comment]
Can we squash & merge this one and have a new release?

**Comment by Barthelemy** [comment]
@knopers8 I let you review.

**Review by knopers8 — APPROVED** [review]

**Pull request merged**